src/core/quicdef.h

```diff
@@ -265,13 +265,25 @@ CXPLAT_STATIC_ASSERT(IS_POWER_OF_TWO(QUIC_MAX_RANGE_DECODE_ACKS), L"Must be powe
 // QUIC_MIN_INITIAL_PACKET_LENGTH in an IPv6 datagram.
 //
 #define QUIC_DPLPMUTD_MIN_MTU                   (QUIC_MIN_INITIAL_PACKET_LENGTH + \
-                                                CXPLAT_MIN_IPV6_HEADER_SIZE     + \
+                                                CXPLAT_MIN_IPV6_HEADER_SIZE + \
                                                 CXPLAT_UDP_HEADER_SIZE)
 
+//
+// The minimum size of the initial packets we send. We pad a little more than
+// the spec-minimum to help with amplification limits for large server
+// certificates. This MUST BE greater than or equal to
+// QUIC_MIN_INITIAL_PACKET_LENGTH.
+//
+#define QUIC_INITIAL_PACKET_LENGTH              1240
+
+CXPLAT_STATIC_ASSERT(QUIC_INITIAL_PACKET_LENGTH >= QUIC_MIN_INITIAL_PACKET_LENGTH, "Packet lengt too small");
+
 //
 // The minimum IP MTU DPLPMTUD will use by default.
 //
-#define QUIC_DPLPMUTD_DEFAULT_MIN_MTU           QUIC_DPLPMUTD_MIN_MTU
+#define QUIC_DPLPMUTD_DEFAULT_MIN_MTU           (QUIC_INITIAL_PACKET_LENGTH + \
```

Comment:
What would you suggest?

Reply:
This is the default value for "min MTU". We also have a default for "max MTU". But these values are also configurable.

Reply:
Is Connection->Settings.MinimumMtu used by mtu_discovery at all? I'm not seeing any evidence of its use, but it's being set by this variable with "DPLPMUTD" (BTW that's another typo) so I must be missing something?

Reply:
OK, I don't have a particularly good name suggestion here and I think it would require understanding this code better so let's just leave the names alone for now.